### DISCUSSION TITLE
Which serving method is good and why? (Pdserving vs Hubserving)

### DISCUSSION THREAD

**Discussion by YC7225** [PaddlePaddle/PaddleOCR#12180] [Q&A]
Which serving method is good and why? (Pdserving vs Hubserving)

**Comment by littletomatodonkey** [comment]
* Hubserving is just for fun and quick experience.
* Pdserving is recommended because its higher performace.

**Comment by YC7225** [comment]
Thank! if I want to just deploy recognition model not the detection model and what are the changes i need to do in the tutorial (pdserving tutorial)?
